wp-content/themes/csisnuclearnetwork/template-parts/newsletter-block-acf.php

```diff
@@ -0,0 +1,30 @@
+<?php
+/**
+ * The about PONI section of the home page,
+ * including information about directors and a team photo
+ *
+ * @link https://developer.wordpress.org/themes/basics/template-hierarchy/
+ *
+ * @package CSIS iLab
+ * @subpackage @package nuclearnetwork
+ * @since 1.0.0
+ */
+
+?>
+
+<?php 
+    $dailyNewsletterName = get_field('daily_newsletter_name');
```

Comment:
We should do a check to make sure that the ACF plugin is installed before you start referencing fields. I would do something like this:

```
if( ! class_exists('ACF') ) {
  return;
}
// your code here
```